File: apps/net/Ch7_Db/Data/Guitar.cs
namespace Ch7_Db.Data
{
    public class Guitar
    {
        public int Id { get; set; }

        public string Name { get; set; }
        public float Price { get; set; }
        public string Img { get; set; }
        public string Style { get; set; }

        public Guitar()
        {
        }

        public Guitar(string name, float price, string img, string style)
        {
            Name = name;
            Price = price;
            Img = img;
            Style = style;
        }
    }
}

File: apps/py/ch08_db/guitary/data/guitar.py
import sqlalchemy as sa
from guitary.data.sqlalchemybase import SqlAlchemyBase


class Guitar(SqlAlchemyBase):
    __tablename__ = 'guitars'

    id: int = sa.Column(sa.Integer, primary_key=True, autoincrement=True)
    name: str = sa.Column(sa.String)
    style: str = sa.Column(sa.String, index=True)
    img: str = sa.Column(sa.String)
    price: float = sa.Column(sa.Float, index=True)
